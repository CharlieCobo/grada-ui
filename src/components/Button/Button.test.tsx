import React from 'react';
import { render } from '@testing-library/react';
import { Button, ButtonProps } from '.';

describe('components/Button', () => {
  const buttonInitProps: ButtonProps = {
    children: 'Click me!',
  };

  it('Should match with snapshot', () => {
    const { container } = render(<Button {...buttonInitProps} />);

    expect(container).toMatchSnapshot();
  });
});
