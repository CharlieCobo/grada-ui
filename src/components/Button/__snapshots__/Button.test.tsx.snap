// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button Should match with snapshot 1`] = `
<div>
  <button
    class="grada-button"
  >
    Click me!
  </button>
</div>
`;
